test/elixir/test/partition_mango_test.exs

```diff
@@ -633,4 +633,31 @@ defmodule PartitionMangoTest do
     %{:body => %{"reason" => reason}} = resp
     assert Regex.match?(~r/Partition must not start/, reason)
   end
+
+  @tag :with_partitioned_db
+  test "partitioned query sends correct errors for sort errors", context do
+    db_name = context[:db_name]
```

Comment:
Current test covers most of change on Partitioned database/index. However, it would be nicer to have one test for `{no_usable_index, missing_sort_index}`

Reply:
There is a mango test in the python tests that exercises that error message.